book/src/getting-started/4-cmake-integration.md

```diff
@@ -67,6 +71,8 @@ This is what generates and compiles the C++ code for our `MyObject` class at bui
 Every Rust source file that uses the `#[cxx_qt::bridge]` macro need to be included in this script.
 In our case, this is only the `src/cxxqt_object.rs` file.
 
+> At this point, you can test your library build by running `QMAKE=/path/to/qmake cargo build` inside the `rust` folder. This should output a static library in `rust/target/debug/qml_minimal.lib` on Windows or `rust/target/debug/qml_minimal.a` on Unix system.
```

Comment:
Do we really need to say this? I'd like to keep the documentation concise.

Reply:
Maybe the second sentence is less useful, but I really think the first one really is.
When you think about it the whole setup is quite complex as it require understanding of cpp, rust, cargo, cmake, qml, qrc, qt. Being able to test at this without bringing further complication is very helpful. 
Having this note here is useful to debug the cargo part of the build without bringing cmake in, finding qt from cmake issues etc...
I aware than when has been done once it is trivial, but on first run, I was happy to just test that part. It worked out of the box my system because I had qmake in the path, I think I would have struggle a bit more or I wouldn't understand everything if I had just runner everything from cmake.

Reply:
I think this suggestion could open a can of worms for users who may think they need to run Cargo separately from CMake. Users might also get confused if running `cargo build` picks up a different version of qmake than CMake finds. If someone wants to play around with Cargo without involving CMake at all, they can run `cargo build -p qml-minimal-no-cmake` as explained in the following chapter.

Reply:
Ok, I haven't read this chapter yet, as I'm interesting in integrating cxx-qt in already existing CMake based application.
I will remove this line because I see your point.

I don't know if people will behave like, but I think they will go either to the building with cmake or cargo page. In the cargo page there is no mention at all of qmake.

Maybe I could duplicate the information about qmake to that other page? But I guess that would be another PR